tests/eas/acceptance.py

```diff
@@ -395,17 +304,27 @@ def test_big_cpus_fully_loaded(self):
 
             self.assertGreater(busy_time, expected_busy_time, msg=msg)
 
-    def test_little_cpus_run_tasks(self):
+    @experiment_test
+    def test_little_cpus_run_tasks(self, experiment, tasks):
         """Offload Migration and Idle Pull: Little cpus run tasks while bigs are busy"""
-        tasks = self.params.keys()
+
         num_offloaded_tasks = len(tasks) / 2
 
-        first_task_finish_time = min(self.end_times.values())
+        end_times = self.calculate_end_times(experiment).values()
+        first_task_finish_time = min(end_times)
```

Comment:
What about having the end_times function to return a:

```python
EndTimes = namedtuple("EndTimes", ['values', 'first', 'last'])
```
where values is populated as `self.calculate_end_times(experiment).values()`, while `first` and `last` are already populated to be min and max of the end_times?

Such a tuple can be returned by the new base class's  method `LisaTest::end_times(self, experiment)`.